pkg/sysinfo/cgroup2_linux.go

```diff
@@ -66,6 +69,25 @@ func newV2(quiet bool, opts *opts) *SysInfo {
 	return sysInfo
 }
 
+func getSwapLimitV2() bool {
+	groups, err := cgroups.ParseCgroupFile("/proc/self/cgroup")
+	if err != nil {
+		return false
+	}
+
+	g := groups[""]
+	if g == "" {
+		return false
+	}
+
+	g = path.Clean(g)
```

Comment:
was the extra `path.Clean()` needed here?

from the docs; https://golang.org/pkg/path/#Join

> Join joins any number of path elements into a single path, separating them with slashes. Empty elements are ignored. **The result is Cleaned.**

Reply:
Maybe it is not needed. I can remove this one.